composer.json

```diff
@@ -17,32 +17,30 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.3|~3.0",
-        "symfony/form": "~2.3|~3.0",
-        "sonata-project/doctrine-orm-admin-bundle": "~2.4",
-        "sonata-project/admin-bundle": "~2.4",
-        "sonata-project/datagrid-bundle": "~2.2"
+        "symfony/framework-bundle": "^2.3 || ^3.0",
+        "symfony/form": "^2.3 || ^3.0",
+        "sonata-project/doctrine-orm-admin-bundle": "3.x-dev@dev",
+        "sonata-project/admin-bundle": "3.x-dev@dev",
+        "sonata-project/datagrid-bundle": "^2.2"
     },
     "require-dev": {
-        "friendsofsymfony/rest-bundle": "~1.1",
-        "jms/serializer-bundle": "~0.11|~1.0",
-        "sonata-project/media-bundle": "~2.2",
-        "symfony/phpunit-bridge": "~2.7|~3.0"
+        "friendsofsymfony/rest-bundle": "^1.1",
+        "jms/serializer-bundle": "~0.11 || ^1.0",
+        "sonata-project/media-bundle": "^2.2",
+        "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "suggest": {
-        "sonata-project/media-bundle": "~2.2"
+        "sonata-project/media-bundle": "^2.2"
```

Comment:
The suggest property should contain text, no version number https://getcomposer.org/doc/04-schema.md#suggest